src/wp-includes/class-wp-script-modules.php

```diff
@@ -18,18 +18,26 @@ class WP_Script_Modules {
 	 * Holds the registered script modules, keyed by script module identifier.
 	 *
 	 * @since 6.5.0
-	 * @var array[]
+	 * @var array<non-empty-string, array<non-empty-string, mixed>>
```

Comment:
non-empty-string is a PHP Stan annotation rather than a phpdocs.

```suggestion
	 * @var array<string, array<string, mixed>>
```